bookmarklet/index.php

```diff
@@ -59,12 +59,14 @@
     </p>
 </div>
 
+<div>
```

Comment:
I see the previous sections declare the id on the `<div>` instead of the inner contents (lines 42, 50).

Should we use `id="bookmarklet-use"` here?